content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,68 @@
+================================================================
+Electronic invoicing (:abbr:`EDI (electronic data interchange)`)
+================================================================
+
+EDI, or electronic data interchange, is the inter-company communication of business documents, such
+as purchase orders and invoices, in a standard format. Sending documents according to an EDI
+standard ensures that the machine receiving the message can interpret the information correctly.
+Various EDI file formats exist.
+
+EDI feature enables automating the administration between companies and might also be required by
+some governments for fiscal control or to facilitate the administration.
+
+Any business that processes large volumes of documents and forms can benefit from an EDI solution to
+increase automation and improve accuracy.
+
+Since July 2022, Odoo supports, among others, the following formats.
+
+.. list-table::
+   :header-rows: 0
+
+   * - Factur-X (CII)
+     - Default format on Odoo (enabled by default)
+   * - Peppol BIS Billing 3.0
+     - For companies which countries are part of the `EAS list
+       <https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/>`_
+   * - E-FFF
+     - For Belgian companies
+   * - NLCIUS
+     - For Dutch companies
+   * - XRechnung UBL
+     - For German companies
+   * - EHF 3.0
+     - Norwegian companies have to use Peppol BIS Billing 3.0 that comply with the EHF 3.0 requirements
```

Comment:
```suggestion
     - Norwegian companies have to use Peppol BIS Billing 3.0 which complies with the EHF 3.0 requirements
```
Maybe we can add the link to the norwegian authority explaining the what and why ? https://anskaffelser.dev/postaward/g3/spec/current/billing-3.0/norway/#_implementation